atsign-foundation/at_server
# notification expiry feature

**- What I did**
Implemented expiry on notification keystore

**- How I did it**
- added ttl and expiresAt to at_notification model
- implemented AtLogType for notification keystore and time based compaction compacts expired entries
**- How to verify it**
- run unit tests in persistence project - at_notification_test and at_notification_keystore_test
- run the notify command by setting ttl. after ttl is elapsed run notify:status:<notification_id>. This should return data:expired
- notify:list command should not display expired notifications
**- Description for the changelog**
Expire notifications based on ttl set in notify command. Default notification expiry to 24 hrs
